docs/user_guide/models/mrvi.md

```diff
@@ -0,0 +1,145 @@
+# MrVI
+
+**MrVI** [^ref1] (Multi-resolution Variational Inference; Python class
+{class}`~scvi.external.MRVI`) is a deep generative model designed for the analysis of large-scale
+single-cell transcriptomics data with multi-sample, multi-batch experimental designs.
+
+The advantages of MrVI are:
+
+- Facilitates both exploratory analysis to divide samples into groups based on cellular and
+    molecular properties and comparative analysis to compare pre-defined groups of samples.
+- Provides single-cell-resolution differential expression and differential abundance estimates.
+- Captures nonlinear, cell-type specific effects of sample-level covariates on gene expression.
+- Scales to large datasets with millions of cells and hundreds of samples.
```

Comment:
Is this unique over other models?

Reply:
no (mostly because the only fair benchmarks are the composition ones), but i'd still like to leave it since by omission it implies we arent scalable (same note is in the other user guides).